api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -494,16 +501,48 @@ public void test_maybeSendAccessExpirationEmail_expiring_1() throws MessagingExc
     accessModuleService.updateCompletionTime(
         dbUser, DbAccessModuleName.RT_COMPLIANCE_TRAINING, willExpireAfter(oneDayPlusSome));
 
-    userService.maybeSendAccessExpirationEmail(dbUser);
+    userService.maybeSendAccessTierExpirationEmails(dbUser);
+
+    verify(mailService)
+        .alertUserAccessTierWarningThreshold(dbUser, 1, expirationTime, REGISTERED_TIER_SHORT_NAME);
+
+    // Control Tier Threshold Email is sent because RT Compliance Training is a requirement for CT
```

Comment:
oh.

I realize this is pretty late in the game, but... is that what we want?

Reply:
I do remember discussing the idea of unified logic (so we only send one email) and rejecting that for the first iteration.  So yeah I think this meets that agreement.

I didn't consider the case where CT is expiring for *strictly RT-related reasons* so I am less happy with this plan now.  But let's keep this as-is, and get back to the more polished version in the near future.

Reply:
Yeah, I don't think I fully considered that case when we first started either. Your suggestion sounds like a good approach to me.